src/libs/actions/Search.ts

```diff
@@ -53,6 +55,22 @@ function search({hash, query, policyIDs, offset, sortBy, sortOrder}: SearchParam
     API.read(READ_COMMANDS.SEARCH, {hash, query, offset, policyIDs, sortBy, sortOrder}, {optimisticData, finallyData});
 }
 
+// TODO_SEARCH: use this function after backend changes.
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+function searchV2(queryString: SearchQueryString) {
+    const queryJSON = buildSearchQueryJSON(queryString);
```

Comment:
It seems like we're already passing the AST here, so no need to try to build it again
```suggestion
```

Reply:
Not sure if I understand. It looks like we pass queryString here not queryJSON.

Reply:
Hmm in my tests I had to remove this to make it work with the backend.